### DISCUSSION TITLE
[简单主页]关于近期通过预设加载简单主页出错的问题

### DISCUSSION THREAD

**Discussion by MFn233** [Meloong-Git/PCL#2974] [讨论]
[原issue见此](https://github.com/Hex-Dragon/PCL2/issues/2963)

经过测试，目前使用“主页预设”打开简单主页会出现如下错误：

![image](https://github.com/Hex-Dragon/PCL2/assets/103323756/c6474bf6-d0be-4e55-8e3b-710ab9665e27)

与cdn提供者讨论后：

![image](https://github.com/Hex-Dragon/PCL2/assets/103323756/ba6a8e5e-3450-47a6-aa1f-cbb015ff5146)

![image](https://github.com/Hex-Dragon/PCL2/assets/103323756/89055634-020e-4197-9c96-7e679257ae32)

于是

在龙猫没有修复替换链接之前，各位使用简单主页可以在“联网更新”一栏填写：

https://gitee.com/mfn233/PCL-Mainpage/raw/main/Custom.xaml

或

https://fastly.jsdelivr.net/gh/MFn233/PCL-Mainpage@latest/Custom.xaml

即可恢复使用（~~不过应该没有人会去用吧~~）

**Comment by MoYuan-CN** [comment]
嘶......

Gitee 用多了仓库可能被按 ` 外链盗用（RAW）` 封掉仓库开源权限

jsdelivr 还是有访问超时现象
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/fe45c4e1-8cd8-43c8-bf9d-72078f79c14a)

**Reply by Deep-Dark-Forest** [reply]
不如试试github fast，也许能撑一段…

**Reply by MoYuan-CN** [reply]
githubfast...
龙猫在哪里说过不想用第三方服务来着

**Reply by Deep-Dark-Forest** [reply]
> githubfast...
> 龙猫在哪里说过不想用第三方服务来着

~~应该不是[#2879](https://github.com/Hex-Dragon/PCL2/pull/2879)~~

**Reply by MoYuan-CN** [reply]
是的

**Reply by MFn233** [reply]
@JingHai-Lingyun 我看来得试试gitcode了。。走投无路了属于是

**Reply by SqrtTan** [reply]
的确可能炸……我早上炸过一次，大概五分钟就好了

**Reply by MoYuan-CN** [reply]
哎 头大

**Comment by WorldHim** [comment]
https://raw.gitmirror.com/MFn233/PCL-Mainpage/main/Custom.xaml
也可以使用

**Reply by MoYuan-CN** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/a5f91950-a4b1-444c-97c8-f2797cc22026)
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/af638c10-e47a-482c-93ee-a205a8726274)
有点超时.. 不过比 jsdelivr 一堆超时要好了..

**Comment by MoYuan-CN** [comment]
看看猫猫咋说
@LTCatt

**Comment by iPodtouch6557** [comment]
顺便在此回复一下
_历史上的今天_ 为啥也不好使了
![image](https://github.com/Hex-Dragon/PCL2/assets/118656946/3b63ad80-23a5-4fc7-b3e4-7d7a9e6aded4)

**Reply by LTCatt** [reply]
@hsuchenghao

**Reply by SqrtTan** [reply]
我这好的……稀奇古怪……

**Reply by MoYuan-CN** [reply]
稀奇古怪.. 我这里也是炸了的
貌似雨云的服务器如果在国外的话用这个 `52chye.cn` 的备案域名好像会炸..

**Reply by hsuchenghao** [reply]
雨云域名的备案爆炸了，等雨云那边fix吧

**Reply by LTCatt** [reply]
在修好之前我先把它从预设里删了，修好了再加回来……
你也可以先用 gitee？
